operator/tests/common/central-cr-assert.yaml

```diff
@@ -30,24 +30,19 @@ collectors:
 - command: retry-kubectl.sh describe pod -n $NAMESPACE -l app=scanner-v4-matcher
 - command: retry-kubectl.sh describe pod -n $NAMESPACE -l app=scanner-v4-db
 # Please keep the above lists in sync with pods-debug.yaml
-timeout: 1500
----
-apiVersion: platform.stackrox.io/v1alpha1
-kind: Central
-metadata:
+- command: retry-kubectl.sh describe central.platform.stackrox.io -n $NAMESPACE stackrox-central-services
+resourceRefs:
+- apiVersion: platform.stackrox.io/v1alpha1
+  kind: Central
   name: stackrox-central-services
-status:
-  conditions:
-  - status: "True"
-    type: Deployed
-  - status: "True"
-    type: Initialized
-  - status: "False"
-    type: Irreconcilable
-  - status: "False"
-    type: ProxyConfigFailed
-  - status: "False"
-    type: ReleaseFailed
+  ref: central_cr
+assertAll:
+- celExpr: "central_cr.status.conditions.filter(c, c.type=='Deployed').exists(c, c.status=='True')"
```

Comment:
In the future  (once the old version is 4.10+) these can be changed back to the non-CEL version, right?

Reply:
Yes, I think so.